packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js

```diff
@@ -65,6 +65,138 @@ describe('plugin-conversation', () => {
       });
     });
 
+    describe('reactions', () => {
+      it('calls the correct method if a reaction2Summary object is passed in', () => {
+        const transform = transforms.find((t) => t.name === 'decryptObject');
+        const transformStub = sinon.stub();
+
+        const ctx = {
+          transform: transformStub
+        };
+        const key = null;
+        const activity = {
+          objectType: 'reaction2Summary'
+        };
+
+        transform.fn(ctx, key, activity);
+
+        assert.equal(transformStub.lastCall.args[0], 'decryptReaction2summary');
```

Comment:
Isn't `transform.fn` async? how do you ensure the assertion is done at the right time if it's not in a callback on the promise resolution?

Reply:
in this case, it does not return a promise. i was following the examples in the tests for microAppInstance, comments, and links